### ISSUE TITLE
Fatal Exception: android.database.sqlite.SQLiteDatabaseLockedException: database is locked (code 5)

### ISSUE THREAD

**Issue by AdrienAlbertini** [lokalise/lokalise-android-sdk#8]
Hi,

After updating Lokalise from 1.0.4 (with local aar) to 1.3.1 with maven (gradle 4.4.1, build tools 27.0.2 and android api 27) we are facing 50% of crashes due to this issue and didn't change anything to lokalise initialization:
 
```
@Override
    public void onCreate() {
        super.onCreate();

   LokaliseSDK.init(BuildVariant.LOKALISE_API_TOKEN, BuildVariant.LOKALISE_PROJECT_ID, this);
   LokaliseSDK.setPreRelease(BuildConfig.DEBUG);
   LokaliseSDK.updateTranslations();
...
```
Fatal Exception: android.database.sqlite.SQLiteDatabaseLockedException: database is locked (code 5)
#################################################################
Error Code : 5 (SQLITE_BUSY)
Caused By : The database file is locked.
	(database is locked (code 5))
#################################################################
       at android.database.sqlite.SQLiteConnection.nativeExecute(SQLiteConnection.java)
       at android.database.sqlite.SQLiteConnection.execute(SQLiteConnection.java:629)
       at android.database.sqlite.SQLiteSession.beginTransactionUnchecked(SQLiteSession.java:319)
       at android.database.sqlite.SQLiteSession.beginTransaction(SQLiteSession.java:298)
       at android.database.sqlite.SQLiteDatabase.beginTransaction(SQLiteDatabase.java:634)
       at android.database.sqlite.SQLiteDatabase.beginTransactionNonExclusive(SQLiteDatabase.java:569)
       at co.lokalise.android.sdk.core.LokaliseDBHelper.updateTranslations(LokaliseDBHelper.java:271)
       at co.lokalise.android.sdk.LokaliseSDK$3.onSuccess(LokaliseSDK.java:350)
       at co.lokalise.android.sdk.library.api.APIController$1.run(APIController.java:253)
       at java.lang.Thread.run(Thread.java:762)

**Comment by DrParanoia** [comment]
Hi! Please contact us through the support chat on our website for further investigation.

**Comment by DrParanoia** [comment]
This should be fixed in version 1.3.3
